apps/qubit/modules/settings/templates/pageElementsSuccess.php

```diff
@@ -50,6 +50,18 @@
             <td><?php echo $form->toggleIoSlider ?></td>
           </tr>
           <tr>
+            <td>
+              <?php echo $form->toggleDigitalObjectMap->label('Digital object map')->renderLabel() ?>
+              <?php if (!$googleMapsApiKeySet): ?>
+                <div id="notice-alerts" class="alert alert-info">
```

Comment:
Should we use our standard `<div class="description">...</div>` instead? See: https://imgur.com/a/5hH4r.